vimish-fold.el

```diff
@@ -261,7 +286,8 @@ This includes fringe bitmaps and faces."
 (defun vimish-fold-unfold ()
   "Delete all `vimish-fold--folded' overlays at point."
   (interactive)
-  (mapc #'vimish-fold--unfold (overlays-at (point))))
+  (save-excursion
```

Comment:
Just to make sure: does `save-excursion` make any difference is this case?

Reply:
It seems like no ... I wonder why I did it. Removed.